api/envoy/extensions/filters/network/tcp_proxy/v3/tcp_proxy.proto

```diff
@@ -259,4 +260,19 @@ message TcpProxy {
 
   // Additional access log options for TCP Proxy.
   TcpAccessLogOptions access_log_options = 17;
+
+  // If set, the specified PROXY protocol TLVs (Type-Length-Value) will be added to the PROXY protocol
```

Comment:
Please add a note that this only has an effect if there's an upstream proxy protocol transport socket.